bootloader/bin/bootstrap.sh

```diff
@@ -292,18 +288,18 @@ exec_init() {
 boot_target() {
   local target="$1"
 
-  echo "moving mounts to newroot"
+  echo "Preparing new root environment"
   mkdir /newroot
   mount $target /newroot
-  #bind mount /dev/console to show systemd boot msgs
+  # Bind mount /dev/console to show systemd boot messages
```

Comment:
Changing the capitalization of comments is pointless. Putting a space between the `#` and the comment is merely a stylistic change and also pointless.

Reply:
Yeah, i guess